WordPress/WordPressUITests/Tests/SupportScreenTests.swift

```diff
@@ -0,0 +1,27 @@
+import XCTest
+
+class SupportScreenTests: XCTestCase {
+
+    override func setUp() {
+        setUpTestSuite()
+
+        LoginFlow.logoutIfNeeded()
+    }
+
+    override func tearDown() {
+        takeScreenshotOfFailedTest()
+        LoginFlow.logoutIfNeeded()
+        super.tearDown()
+    }
+
+    // Test Support Section Loads
+    // From Prologue > continue, tap "help" and make sure Support Screen loads
+    func testSupportScreenLoads() {
+        let supportScreen = PrologueScreen().selectContinue().selectHelp()
+
+        XCTAssert(supportScreen.isLoaded())
+
+        //close SupportScreen modal before teartown
+        supportScreen.closeButton.tap()
```

Comment:
Similarly as above, we don't need the UI test itself to know how to dismiss the support screen, we can move that knowledge in the screen object. Doing so makes the test closer to natural language, and the logic to dismiss the screen reusable.

```suggestion
        supportScreen.dismiss()
```

Moving the logic into the screen object also helps removing comments. It's tempting to use comment to explain what the code does, but it's best to make the code itself clearer, either with better design or naming.

From ["Clean Code" by Uncle Bob](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882):

<img width="500" alt="Screen Shot 2019-10-31 at 9 23 28 am" src="https://user-images.githubusercontent.com/1218433/67903657-4a477380-fbc0-11e9-8f6a-e3c3f210f92a.png">

That's not to say that one should never leave comments in the code. There are many instances in which comments are necessary and add lots of value. Those comment, though, are usually more about the context in which the code sits, the author's rationale for choosing that code, or suggestions for future readers to make improvements.

IMHO, this comment doesn't add much information. It could be removed without impacting a future reader's ability to understand and modify the test.

Reply:
Makes sense to me! I moved closeButton.tap() into a dismiss() function on SupportScreen.

As for the comment, I wanted to give context that we need this dismiss function in the test itself because tearDown() can't deal with modals currently. Most of the UI tests I'm familiar with don't end with a modal, so it's unusual to have another function after the XCTAssert. I've edited the comment to make that more clear.